examples/spec/lcdproc/LCDd.ini

```diff
@@ -0,0 +1,2752 @@
+[]
+mountpoint = LCDd.conf
+;infos/plugins = dump validation enum range ipaddr path network type
+infos/plugins = dump validation enum
```

Comment:
dump would be completely incompatible to current format

Reply:
see reason above, I changed it